app/src/main/res/drawable/ic_logo.xml

```diff
@@ -0,0 +1,24 @@
+      android:fillColor="#EBEBEE"/>
+  <path
+      android:pathData="M12,12m-4.5,0a4.5,4.5 0,1 1,9 0a4.5,4.5 0,1 1,-9 0"
+      android:fillColor="#02A675"/>
+  <path
+      android:pathData="M18.375,18.353C16.745,19.988 14.491,21 12,21C7.029,21 3,16.971 3,12C3,7.029 7.029,3 12,3C14.48,3 16.725,4.003 18.353,5.625L20.474,3.504C18.303,1.339 15.308,0 12,0C5.373,0 0,5.373 0,12C0,18.627 5.373,24 12,24C15.319,24 18.324,22.652 20.496,20.474L18.375,18.353Z"
+      android:strokeAlpha="0.4"
+      android:fillColor="#02A675"
+      android:fillType="evenOdd"
+      android:fillAlpha="0.4"/>
+  <path
+      android:pathData="M24,12m-4.5,0a4.5,4.5 0,1 1,9 0a4.5,4.5 0,1 1,-9 0"
+      android:fillColor="#F5B100"/>
+  <path
+      android:pathData="M36,12m-4.5,0a4.5,4.5 0,1 1,9 0a4.5,4.5 0,1 1,-9 0"
+      android:fillColor="#E85151"/>
+</vector>
```

Comment:
We already have `ic_app.xml`, please replace this with `ic_app` + `app_name` string. Just in case we renamed the app in the future.